style.css

```diff
@@ -0,0 +1,9 @@
+#navbar{
+    position:fixed;
+    top:0;
+}
```

Comment:
This works but 💡 remember 💡  to check linter errors such as one space between '#navbar' and ' { ', also only use two spaces indentation for 'position' and 'top'. Linters always help you make your code cleaner and easier to understand and review by others. 😄